externals/llvm-external-projects/torch-mlir-dialects/test/Conversion/StablehloToTcp/stablehlo.mlir

```diff
@@ -0,0 +1,11 @@
+// RUN: torch-mlir-dialects-opt <%s -convert-stablehlo-to-tcp -split-input-file | FileCheck %s
+
+// CHECK-LABEL: func.func @tanh(
+// CHECK-SAME:                %[[ARG0:.*]]: tensor<f32>) -> tensor<f32> {
+// CHECK:         %[[TANH:.*]] = tcp.tanh %[[ARG0]] : tensor<f32>
+// CHECK:         return %[[TANH]] : tensor<f32>
+// CHECK:       }
+func.func @tanh(%arg0: tensor<f32>) -> tensor<f32> {
```

Comment:
minor nit: typically, we try to add tests with dynamic shapes in them, so that we are ensure that for all ops.

Reply:
Thank you! Updated the test to be consistent with tests in the Linalg lowering.